CHANGELOG.adoc

```diff
@@ -2,6 +2,25 @@
 
 = Changelog
 
+== v1.10.0 (2019-02-28)
+
+* Automatically detect when the ES operator is available (https://github.com/jaegertracing/jaeger-operator/pull/239[#239])
+* Adjusted logs to be consistent across the code base (https://github.com/jaegertracing/jaeger-operator/pull/237[#237])
+* Fixed deployment of Elasticsearch via its operator (https://github.com/jaegertracing/jaeger-operator/pull/234[#234])
```

Comment:
nit: some of the lines are related to fixes for features introduced in this release - I would leave them out to make the changelog more readable

Reply:
As we talked on IRC, for a next release we'll aim to have it more structured as we have for Jaeger's main repo